docs/stats.md

```diff
@@ -0,0 +1,30 @@
+## accessing stats
+
+stats can be accessed via 
+1. `pumactl`
+2. Puma.stats when in non-worker mode
+3. Puma.stats when in worker mode (should not be invoked from worker process, right?)
+
+
+## meaning of stats
+
+* started_at: when puma was started
+* phase: ??
```

Comment:
Following how phased restarts work in `cluster.rb` is probably the best resource for this. Off the top of my head, I think the cluster itself has a `phase`, and then each individual worker has a phase as well. Phase starts at 0 for both, but when you issue a phased restart (described in the [restart docs](https://github.com/puma/puma/blob/e2a15675a37bd25184fa7ad8343d23577464a5a0/docs/restart.md)), the phase of the cluster is incremented. Then, workers started in the old phase die off while new workers are spun up, recording their phase as the cluster's current phase. Monitoring the phases of the workers can be used, for example, to see if a phased restart operation was successful or not.

Reply:
Looking at the code I wasn't able to confidently infer what the different phases were and how to describe them. Maybe someone who knows could flesh that out in https://github.com/puma/puma/blob/master/docs/restart.md and then we can link to it in the stats doc